app/controllers/v2/content_items_controller.rb

```diff
@@ -18,17 +18,30 @@ def index
     def linkables
       # Base path is returned to facilitate rummager indexing.
       # This can be removed once link updates are picked up by rummager from the message bus.
-      render json: Queries::GetContentCollection.new(
-        document_type: query_params.fetch(:document_type),
-        fields: %w(
-          title
-          content_id
-          publication_state
-          base_path
-          internal_name
-        ),
-        pagination: NullPagination.new
-      ).call
+      document_type = query_params.fetch(:document_type)
+
+      last_update_to_document_type = ContentItem
+        .where(document_type: [document_type, "placeholder_#{document_type}"])
+        .order('updated_at ASC')
+        .select("updated_at")
+        .last
+        .updated_at
```

Comment:
To add to this further, my preference would be to use a combination of [`ActionController::Base#stale?`](http://api.rubyonrails.org/classes/ActionController/ConditionalGet.html#method-i-stale-3F) and [`Rack::Cache`](http://rtomayko.github.io/rack-cache/) with the memcached backend to implement this.